### DISCUSSION TITLE
Invoke AWS Lambda and get results after it finished

### DISCUSSION THREAD

**Discussion by AntonFedo** [artilleryio/artillery#1698] [Artillery]
Hi everyone! I have a question about performance testing AWS Lambda.
Am I able to invoke AWS Lambda from artillery?
I would like to invoke lambda wait until it finished and than display results of this invoking (Min time, max time and other statistics).
I can not find any information about configuration of tests of AWS Lambda in documentation

**Comment by hassy** [comment]
Hi @AntonFedo 👋 

There's this engine - https://www.npmjs.com/package/artillery-engine-lambda - but it hasn't been updated in a while and may not work with Artillery v2. Worth giving it a go though, and if it doesn't work, it should be pretty straightforward to update it for Artillery v2.
